dockerfiles/worker.Dockerfile

```diff
@@ -87,7 +87,8 @@ RUN pip install -r singa_auto/kafka/requirements.txt
 COPY singa_auto/advisor/requirements.txt singa_auto/advisor/requirements.txt
 RUN pip install -r singa_auto/advisor/requirements.txt
 
-RUN pip install singa-auto==0.1.9
+RUN pip install singa-auto==0.2.1
+RUN pip install singa-easy==0.2.0
```

Comment:
I think singa-easy can be installed like other dependent libs, such as sklearn.
Then we do not need to install it together with singa-auto.
For one task, if it depends on singa-easy, then install it when the job is launched.